Upstream-Status: Inappropriate [OpenXT]

tboot's CFLAGS_WARN includes -Werror. GCC 15 adds warnings that
then fail the build. Keep them as warnings.

--- a/Config.mk	2026-09-10 17:44:40.663604054 -0400
+++ b/Config.mk	2026-09-10 17:45:00.811603931 -0400
@@ -43,6 +43,9 @@
 	            -Wextra -Winit-self -Wswitch-default -Wunused-parameter \
 	            -Wwrite-strings \
 	            $(call cc-option,$(CC),-Wlogical-op,) \
+	            -Wno-error \
+	            -Wno-error=array-parameter \
+	            -Wno-error=unterminated-string-initialization \
 	            -Wno-missing-field-initializers -Wno-address-of-packed-member
 
 AS         ?= as
